cmk/base/plugins/agent_based/fortigate_antivirus.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+# author: Oguzhan Cicek, OpenSource Security GmbH - oguzhan(at)os-s.de
+
+#Example Input:
+#.1.3.6.1.2.1.1.1.0 test
+#.1.3.6.1.2.1.1.2.0 .1.3.6.1.4.1.12356.101.1.15000
+#[...]
+#.1.3.6.1.4.1.12356.101.8.2.1.1.1.102 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.1.103 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.1.104 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.1.105 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.1.106 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.2.102 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.2.103 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.2.104 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.2.105 0
+#.1.3.6.1.4.1.12356.101.8.2.1.1.2.106 0
+#[...]
+
+#Example GUI Output:
+#OK	FortiGate AntiVirus Info: 0
+#       Viruses per second: detected: 0.00, blocked: 0.00
+#OK	FortiGate AntiVirus Info: 1
+#       Viruses per second: detected: 0.00, blocked: 0.00
+#OK	FortiGate AntiVirus Info: 2
+#       Viruses per second: detected: 0.00, blocked: 0.00
+#OK	FortiGate AntiVirus Info: 3
+#       Viruses per second: detected: 0.00, blocked: 0.00
+#OK	FortiGate AntiVirus Info: 4
+#       Viruses per second: detected: 0.00, blocked: 0.00
+
+from .agent_based_api.v1.type_defs import (
+    DiscoveryResult,
+    CheckResult,
+    StringTable,
+)
+from .agent_based_api.v1 import (
+    register,
+    get_rate,
+    Service,
+    startswith,
+    Metric,
+    SNMPTree,
+    check_levels,
+    get_value_store,
+)
+from typing import List, Dict, Mapping, Tuple
+import time
+
+Section = Dict[str, Dict[str, int]]
+
+
+def parse_fortigate_antivirus(string_table: List[StringTable]) -> Section:
+    parsed = {}
+    for i in range(len(string_table[0])):
+        parsed.update({
+            str(i): {
```

Comment:
One question: Would it make sense to use the OID ending (102, 103, ...) as item, or is this information not useful here?

Reply:
In case it is useful, I can take care of it while merging the PR.

Reply:
you might be right using the oid end would be better than using some incremental number.
yes the check creates one service per virtual domain. i just updated both fortigate_antivirus and fortigate_ips

Reply:
Very good, thanks for the fast reply. Is the item (aka OIDEnd()) then the number of the virtual domain? Or how would you call it? I am asking because I want to add this to the check manual.

Reply:
The MIB of the Fortinet FortiGate says: "Internal virtual domain index used to uniquely identify rows in this table. This index is also used by other tables referencing a virtual domain."